components/server/src/auth/generic-auth-provider.ts

```diff
@@ -145,11 +145,20 @@ export class GenericAuthProvider implements AuthProvider {
     protected readAuthUserSetup?: (accessToken: string, tokenResponse: object) => Promise<AuthUserSetup>;
 
     authorize(req: express.Request, res: express.Response, next: express.NextFunction, scope?: string[]): void {
-        const handler = passport.authenticate(this.getStrategy() as any, {
-            ...this.defaultStrategyOptions,
-            ...{ scope },
+        req.session.cookie.sameSite = "lax";
```

Comment:
setting to `lax` before redirecting to GH/GL

Reply:
:point_up: that should be part of a bigger comment explaining what we aim for, and pointing to the related parts of the code further down/in session-handler. :pray:

Reply:
done